OpenEdX/AppDelegate.swift

```diff
@@ -70,6 +75,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]
     ) -> Bool {
         if let config = Container.shared.resolve(ConfigProtocol.self) {
```

Comment:
Although it's not part of the underlying PR, but as we are, how about fixing it as well, like the below code for facebook can return false, whereas the method should have been called for Google or microsoft. It would be nice if we see if the URL is for facebook only then it will be returning the status.
```
if config.facebook.enabled {
            ApplicationDelegate.shared.application(
                app,
                open: url,
                sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
                annotation: options[UIApplication.OpenURLOptionsKey.annotation]
            )
        }
```